eng/Signing.props

```diff
@@ -1,16 +1,41 @@
 <Project>
 
   <ItemGroup>
-    <!-- Do not sign non-shipping packages -->
+    <!-- Do not sign non-shipping packages with doing in-build signing -->
     <ItemsToSign Remove="$(ArtifactsNonShippingPackagesDir)**\*.nupkg" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(PostBuildSign)' == 'true'">
+    <ItemsToSignPostBuild Include="$(ArtifactsShippingPackagesDir)*.zip" />
```

Comment:
don't wixpack.zips usually go to nonshipping?

Reply:
Will verify

Reply:
The arcade defaults include these already.